### PR TITLE
More e2e tests - User Status

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Adds more tests for User Status
- Adds missing classes and proptypes

### CODE REVIEW

File: test/e2e/specs/02_admin.js [hunk 1]
```diff
@@ -1,56 +1,43 @@
...
-      .setValue('@emailInput', admin.email)
-      .setValue('@passwordInput', admin.password)
-      .waitForElementVisible('@signInButton')
-      .click('@signInButton');
-
-    adminPage
-      .waitForElementVisible('@moderationContainer');
+    adminPage.navigate();
+    adminPage.expect.section('@login').to.be.visible;
+    adminPage.section.login.login(admin);
```
Comment (okbel) [inline test/e2e/specs/02_admin.js]: Why did you move the login command inside the login section? It should be a general command, IMO. `admin.login(userData)` => To login inside the admin-side no matter where you are.

Comment (alinhle) [reply test/e2e/specs/02_admin.js]: Here is my reasoning:
- I'm splitting the pages into logical parts, e.g. the login section is only available when you are not logged in.
- I'm avoiding using `navigate` as much as possible to simulate a user clicking through the page, also decreases testing time and increases stability of the tests.
- The commands I added are stateless building blocks of user interaction that you can do at a certain part of the page (without calling `navigate`). Think of it as more lower level commands.
- We can provide a top level command called `navigateAndLogin` that calls the lower level commands for you.

Comment (alinhle) [reply test/e2e/specs/02_admin.js]: I'll add a `navigateAndLogin` method.

File: test/e2e/page_objects/admin.js [hunk 2]
```diff
@@ -14,6 +14,12 @@ module.exports = {
       this.expect.section('@drawer').to.be.visible;
       return this.section.drawer;
     },
+    goToModerate() {
+      this
+        .click('@moderateNav')
+        .expect.section('@moderate').to.be.visible;
+      return this.section.stories;
```
Comment (alinhle) [inline test/e2e/page_objects/admin.js]: `this.section.moderate`

File: client/coral-admin/src/components/ActionsMenuItem.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import cn from 'classnames';
 import {MenuItem} from 'react-mdl';
 import PropTypes from 'prop-types';
 import styles from './ActionsMenu.css';
-import camelCase from 'lodash/camelcase';
+import camelCase from 'lodash.camelcase';
```
Comment (alinhle) [inline client/coral-admin/src/components/ActionsMenuItem.js]: Why the change?

Comment (alinhle) [reply client/coral-admin/src/components/ActionsMenuItem.js]: If it didnt work try `lodash/camelCase`

Comment (okbel) [reply client/coral-admin/src/components/ActionsMenuItem.js]: Because it wasn't working on our CI
```
ModuleNotFoundError: Module not found: Error: Can't resolve 'lodash/camelcase' in '/home/ubuntu/talk/client/coral-admin/src/components'
    at factoryCallback (/home/ubuntu/talk/node_modules/webpack/lib/Compilation.js:260:39)
    at factory (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:247:20)
    at resolver (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:65:21)
    at asyncLib.parallel (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:138:21)
```

File: test/e2e/page_objects/admin.js (hunk 2 above)
Comment (okbel) [reply test/e2e/page_objects/admin.js]: 👍

File: client/coral-admin/src/components/ActionsMenuItem.js (hunk 3 above)
Comment (alinhle) [reply client/coral-admin/src/components/ActionsMenuItem.js]: did you try `lodash/camelCase`?

Comment (alinhle) [reply client/coral-admin/src/components/ActionsMenuItem.js]: Notice the case

Comment (okbel) [reply client/coral-admin/src/components/ActionsMenuItem.js]: On my end both are working. I can give it a shot.

Comment (alinhle) [reply client/coral-admin/src/components/ActionsMenuItem.js]: Yeah some filesystems are case insensitive

**Review by alinhle — APPROVED** [review]
